package.json

```diff
@@ -36,6 +36,7 @@
     "@aws-sdk/client-s3": "^3.279.0",
     "@aws-sdk/credential-providers": "^3.281.0",
     "@stencila/encoda": "0.121.1",
+    "@stencila/schema": "^1.18.0",
```

Comment:
We should check with stencila that it's ok for us to use this for now. The archived the repo in January. 

https://github.com/stencila/schema#%EF%B8%8F-moved

Reply:
I reckon that's because they carry on work on the rust version that no longer depends on this repo. As I mentioned in another comment - I think we'd be better to work towards a stencila-schema-agnostic server if we can, but while we're still using the typescript version it should be fine.